modules/installation-configuration-parameters.adoc

```diff
@@ -1277,8 +1277,13 @@ The value can also be the name of an existing Glance image, for example `my-rhco
 You can use this property to exceed the default persistent volume (PV) limit for {rh-openstack} of 26 PVs per node. To exceed the limit, set the `hw_scsi_model` property value to `virtio-scsi` and the `hw_disk_bus` value to  `scsi`.
 
 You can also use this property to enable the QEMU guest agent by including the `hw_qemu_guest_agent` property with a value of `yes`.
-|A list of key-value string pairs. For example, `["hw_scsi_model": "virtio-scsi", "hw_disk_bus": "scsi"]`.
-
+|A set of string properties. For example, `{"hw_scsi_model": "virtio-scsi", "hw_disk_bus": "scsi"}`.
+// For QE:
+// Would a map of pairs be preferable/better suppported? E.g.:
```

Comment:
Not a QE, but I can say that while it is identically supported, it would be far less surprising to see the properties written in yaml style like you show below. I initially assumed that it was not possible due to space constraints

Reply:
That was also my inclination. 

The table styling on the docs site makes me really reach for anything else, but I'll swap that in. I was leaning there, anyway. 👍